src/pages/Landing/components/StudentTicket/Report/Report.module.scss

```diff
@@ -0,0 +1,182 @@
+@import 'sass/variables';
+// @import 'sass/typography';
+.Prob_Report{
```

Comment:
Apply upper camel case for name classes. Eg ` ProbReport`

Reply:
ok